scripts/openmc-ace-to-hdf5

```diff
@@ -0,0 +1,205 @@
+                    'lists ACE libraries')
+parser.add_argument('--xsdir', help='MCNP xsdir file that lists '
+                    'ACE libraries')
+parser.add_argument('--xsdata', help='Serpent xsdata file that lists '
+                    'ACE libraries')
+parser.add_argument('--libver', choices=['earliest', 'latest'],
+                    default='earliest', help="Output HDF5 versioning. Use "
+                    "'earliest' for backwards compatibility or 'latest' for "
+                    "performance")
+args = parser.parse_args()
+
+if not os.path.isdir(args.destination):
+    os.mkdir(args.destination)
+
+# If the --xml argument was given, get the list of ACE libraries directory from
+# <ace_table> elements within the specified cross_sections.xml file
+ace_libraries = []
+if args.xml is not None:
+    tree = ET.parse(args.xml)
+    root = tree.getroot()
+    if root.find('directory') is not None:
+        directory = root.find('directory').text
+    else:
+        directory = os.path.dirname(args.xml)
+
+    for ace_table in root.findall('ace_table'):
+        path = os.path.join(directory, ace_table.attrib['path'])
+        if path not in ace_libraries:
+            ace_libraries.append(path)
+
+elif args.xsdir is not None:
+    # Find 'directory' section
+    lines = open(args.xsdir, 'r').readlines()
+    for index, line in enumerate(lines):
+        if line.strip().lower() == 'directory':
+            break
+    else:
+        raise IOError("Could not find 'directory' section in MCNP xsdir file")
+
+    # Handle continuation lines indicated by '+' at end of line
+    lines = lines[index + 1:]
+    continue_lines = [i for i, line in enumerate(lines)
+                      if line.strip().endswith('+')]
+    for i in reversed(continue_lines):
+        lines[i] += lines[i].strip()[:-1] + lines.pop(i + 1)
+
+    # Create list of ACE libraries
+    for line in lines:
+        words = line.split()
+        if len(words) < 3:
+            continue
+
+        path = os.path.join(os.path.dirname(args.xsdir), words[2])
+        if path not in ace_libraries:
+            ace_libraries.append(path)
+
+elif args.xsdata is not None:
+    with open(args.xsdata, 'r') as xsdata:
+        for line in xsdata:
+            words = line.split()
+            if len(words) >= 9:
+                path = os.path.join(os.path.dirname(args.xsdata), words[8])
+                if path not in ace_libraries:
+                    ace_libraries.append(path)
+
+else:
+    ace_libraries = args.libraries
+
+nuclides = {}
+library = openmc.data.DataLibrary()
+
+for filename in ace_libraries:
+    # Check that ACE library exists
+    if not os.path.exists(filename):
+        warnings.warn("ACE library '{}' does not exist.".format(filename))
+        continue
+
+    lib = openmc.data.ace.Library(filename)
+    for table in lib.tables:
+        name, xs = table.name.split('.')
+        if xs.endswith('c'):
```

Comment:
I would consider moving this to a function as `handleContinuousEnergyNeutronData(...)` which makes the code modular, simpler to read, and allows the author to provide a function-level comment